css-view-transitions-1/Overview.bs

```diff
@@ -1538,6 +1538,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 								backdrop-filter: <var>backdropFilter</var>;
 							}
 						}
+
+						@media (prefers-reduced-motion) {
```

Comment:
The thinking here was to reset the `animation` property on the group pseudo instead of modifying the keyframe. So if the author does want to reinstate the size/position animation they'll have access to the keyframe. But I didn't realize that other properties are part of this keyframe too.

And actually more will be added with the new capture mode.